Makefile

```diff
@@ -105,6 +105,26 @@ deb: $(ARTIFACT_PACKAGE)
 		tidb-deb-builder:${TAG} fakeroot dpkg-deb --build ${ARTIFACT_PACKAGE} /build/dist
 	rm -rf ${ARTIFACT_PACKAGE}
 
+.PHONY: brew
+brew: $(ARTIFACT_DIR) source
```

Comment:
I think for osx we should provide two types of installation:
1. install by source. Homebrew call this project and compile it at localhost then install.
2. install by binary. we build a .tar.gz file contains all the bin&conf and save it on the file server, brew can depend on this tarball for installation.

Reply:
We can use `make brew` to install by source and add `make brew-bottle` to install by binary. I will add the `brew-bottle` target later.

What do you think?

Reply:
I think we should set up a process to provide bottles *before* we try adding tikv to homebrew. The rust toolchain is so big and the build process is so time-consuming, it doesn't seem reasonable to have users go through that to install TiKV.